### PR TITLE
pokemon toggle fixed

### PR DESCRIPTION
**Pull request by hgentry**
## Description

The Pokemon toggle wasn't including lured Pokemon.
## Motivation and Context

#2329 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hgentry** [comment]
@invisiblek hi

**Comment by invisiblek** [comment]
woah, what's with all the changes here?

**Comment by hgentry** [comment]
That's /dist/js/map.js that has all the seemingly random changes. They're mostly whitespace removal, it seems. It's what was generated by grunt. The other changed file is my map.js :)

**Comment by invisiblek** [comment]
I just regen'd via grunt a few hours ago and it didn't do that...

Also - I'm voting for passing an array as buildSwitchChangeListener's second parameter and looping through it as needed. 

https://gist.github.com/invisiblek/b0fafbeddaaf87891029ef5ffe8d82b8

**Comment by hgentry** [comment]
new stuff in

**Comment by hgentry** [comment]
Rebased

**Comment by hgentry** [comment]
Rebased again @invisiblek

File: static/map.js [hunk 1]
```diff
@@ -1178,26 +1178,27 @@ $(function() {
...
-  function buildSwitchChangeListener(data, data_type, storageKey) {
-    return function() {
-      Store.set(storageKey, this.checked);
-      if (this.checked) {
-        updateMap();
-      } else {
-        $.each(data[data_type], function(key, value) {
-          data[data_type][key].marker.setMap(null);
+  
+function buildSwitchChangeListener(data, data_type, storageKey) {
```
Comment (invisiblek) [inline static/map.js]: should have left this whitespace alone, someone just went and cleaned up this file, it should be indented as it was (its all part of the function starting on line 1101)

Comment (hgentry) [inline static/map.js]: done

**Comment by hgentry** [comment]
Ready to go

File: static/map.js [hunk 2]
```diff
@@ -1179,26 +1179,27 @@ $(function() {
       });
     }
   }, 1000);
-
-
```
Comment (invisiblek) [inline static/map.js]: I hate being such a whitespace nazi....but andrewda literally just went through this file 2 commits ago and fixed all our whitespace problems

File: static/map.js [hunk 3]
```diff
@@ -1179,26 +1179,27 @@ $(function() {
...
+          });
+          data[d_type] = {}
         });
-        data[data_type] = {}
       }
-    }
+    });
   }
 
+
```
Comment (invisiblek) [inline static/map.js]: extra

**Comment by hgentry** [comment]
Good to go

**Pull request merged**